magefile.go

```diff
@@ -289,6 +289,16 @@ func Appveyor() error {
 	if err != nil {
 		return err
 	}
+	
+	err_x86 := Builder{
+		goos:       "windows",
+		arch:       "386",
+		extra_tags: " release ",
```

Comment:
ok - it fails now because there are no 32 bit C compilers.

You can add disable_cgo: true, to this to make the build work but you dont get yara and some WMI plugins (we need to build it with a C compiler).